### PR TITLE
Twitter Automation Code Added

### PR DESCRIPTION
**Pull request by sharur7**
### Pre-requisites:
Must have a Twitter account and Twitter developer account.
Need to add your access keys to use twitter APIs

### Working:
Just run it and it will **like** and **retweet** tweets of the value which you have assigned to the "search".

This was the issue response for #94

### CODE REVIEW

**Comment by Neilblaze** [comment]
Amazing work @sharur7 ! :tada: 

Before I approve changes to the PR, I will request you the following changes mentioned below.

- Add README (guide) along with this inside the folder same as your `<filename>`
- Maintain proper [Contribution protocols](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) (ignore if you're already following it!) & [good practices](https://gist.github.com/sloria/7001839)!

Rest LGTM :+1: 
> + nice & clean code :smile:

**Comment by sharur7** [comment]
@Neilblaze Can you please help me my image is not showing up in the README.md file.

**Comment by Neilblaze** [comment]
> @Neilblaze Can you please help me my image is not showing up in the README.md file.

Change it to this - 
```
<p align="center">
  <img src="twitter image.PNG"></p>
```

Sharvari, I'll recommend you a small piece of advice. **Don't** insert any whitespace between any image name. It is considered as a bad practice while working with UTF in markdown.

If the change that I just suggested don't works, just rename the jpg image to twitter.png or image.png & replace the name in the `img` tag!

**Comment by sharur7** [comment]
@Neilblaze Thank You! It Worked... Is there any change that I should do more?

File: System-Automation-Scripts/Twitter Automation.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (Neilblaze) [inline System-Automation-Scripts/Twitter Automation.py]: Delete the two empty lines at the top

**Review by Neilblaze — CHANGES_REQUESTED** [review]

File: System-Automation-Scripts/Twitter Automation.py (hunk 1 above)
Comment (sharur7) [reply System-Automation-Scripts/Twitter Automation.py]: Yep Done!

Comment (sharur7) [reply System-Automation-Scripts/Twitter Automation.py]: Should I squash all commits?

Comment (Neilblaze) [reply System-Automation-Scripts/Twitter Automation.py]: Yes, that sounds better! :smiley:

Comment (Neilblaze) [reply System-Automation-Scripts/Twitter Automation.py]: Yep, that sounds better! :smiley:

**Review by Neilblaze — APPROVED** [review]
LGTM!
Awesome work @sharur7 ! :tada:

**Comment by Neilblaze** [comment]
Code quality looking good :+1: 
@ankitdobhal if possible, resolve the conflicts please!

**Pull request merged**